gst_plugins/python/pyfunc.py

```diff
@@ -145,56 +150,110 @@ def do_set_property(self, prop: GObject.GParamSpec, value: Any):
 
     def do_start(self):
         """Do on plugin start."""
-        self.pyfunc = PyFunc(
-            module=self.module,
-            class_name=self.class_name,
-            kwargs=json.loads(self.kwargs) if self.kwargs else None,
-            dev_mode=self.dev_mode,
-        )
+        # pylint: disable=broad-exception-caught
+        try:
+            kwargs = json.loads(self.kwargs)
+        except Exception as exc:
+            self.handle_non_fatal_error(
```

Comment:
Why is an error in the arguments not fatal?

Reply:
It can be possible for the application to run without any kwargs in a given pyfunc instance, if the instance doesn't use them. If the kwargs end up being mandatory during pyfunc `init`/`load_user_code`/`process_buffer` we will see appropriate error.